velox/docs/functions/presto/binary.rst

```diff
@@ -26,6 +26,14 @@ Binary Functions
 
     Decodes binary data from the hex encoded ``string``.
 
+.. function:: from_ieee754_32(binary) -> real
+
+    Decodes the 32-bit big-endian ``binary`` in IEEE 754 single-precision floating-point format.
```

Comment:
Would it be possible to clarify what happens if input is shorter / longer than 32 bits?

Reply:
@Yuhta Thanks for the review.
Added a line in the docs. PTAL.

Reply:
I think it actually throws user error (`VeloxUserError`)

Reply:
You are right, it does throw a `VeloxUserError`.
I tried to keep it generic as `VeloxUserError` is also a `VeloxException`, but changed it to `VeloxUserError`.
Please take another look.